### PR TITLE
Event_info layout 【Backlog #7】

### PR DESCRIPTION
**Pull request by kokioba95**
## Description / Purpose (checkpoints)
This PR, only the layout confirmation of event_info page.

Fixes issue: < https://docs.google.com/spreadsheets/d/1Ear__g58irVlaI_tpzWUkf__JRcBz8u2d2DdRYKMiDk/edit#gid=1270402250 > <7>

## How To Test This PR
1. Go /event_info (Ex.http://localhost:8086/event_info)
2 .Check the layout
 
 **Commands**
None (layout Only)

 **Screenshots**
 - < https://drive.google.com/open?id=1UX_at7VJu0OeH5Mb7Eo7ULK3hSw7Ws-W >

### CODE REVIEW

File: app/Http/Controllers/EventController.php [hunk 1]
```diff
@@ -10,4 +10,12 @@ public function list()
     {
         return view('event_list');
     }
+<<<<<<< Updated upstream
```
Comment (AubreyV) [inline app/Http/Controllers/EventController.php]: Please remove this, this will cause errors

File: app/Http/Controllers/EventController.php [hunk 2]
```diff
@@ -10,4 +10,12 @@ public function list()
...
 }
+=======
```
Comment (AubreyV) [inline app/Http/Controllers/EventController.php]: this too

File: app/Http/Controllers/EventController.php [hunk 3]
```diff
@@ -10,4 +10,12 @@ public function list()
...
+    public function info()
+    {
+        return view('event_info');
+    }
+}
+>>>>>>> Stashed changes
```
Comment (AubreyV) [inline app/Http/Controllers/EventController.php]: here also

File: routes/web.php [hunk 4]
```diff
@@ -15,5 +15,10 @@
     return view('welcome');
 });
 Auth::routes();
+<<<<<<< Updated upstream
```
Comment (AubreyV) [inline routes/web.php]: Fix conflicts on this file.

File: app/Http/Controllers/EventController.php (hunk 1 above)
Comment (kokioba95) [reply app/Http/Controllers/EventController.php]: The code was fixed.

File: app/Http/Controllers/EventController.php (hunk 2 above)
Comment (kokioba95) [reply app/Http/Controllers/EventController.php]: The code was fixed.

File: routes/web.php (hunk 4 above)
Comment (kokioba95) [reply routes/web.php]: The code was fixed.

File: app/Http/Controllers/EventController.php (hunk 3 above)
Comment (kokioba95) [reply app/Http/Controllers/EventController.php]: The code was fixed.

**Pull request merged**